support/src/main/java/org/cardanofoundation/lob/app/support/spring_audit/CommonDateOnlyLockableEntity.java

```diff
@@ -0,0 +1,59 @@
+package org.cardanofoundation.lob.app.support.spring_audit;
+
+import static jakarta.persistence.TemporalType.TIMESTAMP;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.MappedSuperclass;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+import jakarta.persistence.Transient;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import org.javers.core.metamodel.annotation.DiffIgnore;
+
+@Setter
+@Getter
+@MappedSuperclass
+@NoArgsConstructor
+public abstract class CommonDateOnlyLockableEntity {
+
+    @Temporal(TIMESTAMP)
+    @Column(name = "created_at")
+    @CreatedDate
+    @DiffIgnore
+    protected LocalDateTime createdAt;
+
+    @Temporal(TIMESTAMP)
+    @Column(name = "updated_at")
+    @LastModifiedDate
+    @DiffIgnore
+    protected LocalDateTime updatedAt;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "locked_at")
+    protected LocalDateTime lockedAt;
```

Comment:
Generally for nullables I try to use Optional<LocalDateTime> getLockedAt()

Reply:
In other words you prefer the solution with nulls? Wanna chat a bit at least about my rationale for it?